methods/build.js

```diff
@@ -1,55 +1,58 @@
-const runshell = require('runshell');
+const pMap = require('p-map');
+const qs = require('qs');
 
-module.exports = function (config, settings, data) {
+
+module.exports = async function (config, settings, data) {
   const server = this;
   if (config.length === 0) {
     server.log(['github', 'debug'], { message: 'no matches, skipping', data });
   }
 
   const buildService = async function(item) {
     let before = data.before;
+
     if (item.config.alwaysBuild) {
       before = null;
     }
 
+    const envVars = {
+      USER: data.user,
+      REPO: data.repo,
+      BRANCH: data.branch || data.tag,
+      TOKEN: settings.githubToken,
+      IMAGE_NAME: item.image,
+      DOCKERFILE: item.config.dockerfile || 'Dockerfile',
+      BEFORE: before || '',
+      MONOREPO: item.config.monorepo || false,
+      CONTEXT: item.config.context || '.',
+      DEBUG: 1,
+    };
+
+    if (item.config.hook) {
+      envVars.WEBHOOK = item.config.hook.urls;
+      envVars.WEBHOOK_DATA = qs.stringify(item.config.hook.payload);
+    }
+
+    if (item.config.monorepo && item.config.hook) {
```

Comment:
mono repo hook really is designed for something else, so I think we should name the config value monorepoHook

Reply:
also, until we have docker-app set up, we need to fire off hooks for each build instead of just at the end, so we'll need to be able to set up both hooks

Reply:
So - put the old hooks code back for now that was removed below right?